bundles/org.openhab.binding.mybmw/README.md

```diff
@@ -507,32 +512,38 @@ If these preconditions are fulfilled proceed with the fingerprint generation.
 
 #### Generate Debug Fingerprint
 
-<img align="right" src="./doc/DiscoveryScan.png" width="400" height="350"/>
+Login to the openHAB console and use the `mybmw fingerprint` command.
 
-First [enable debug logging](https://www.openhab.org/docs/administration/logging.html#defining-what-to-log) for the binding.
+Fingerprint information on your account and vehicle(s) will show in the console and can be copiedfrom there.
+A zip file with fingerprint information for your vehicle(s) will also be generated and put in the `mybmw`
+folder in your home directory.
+This fingerprint information is valuable for the developers to better support your vehicle in thesoftware.
 
-```shell
-log:set DEBUG org.openhab.binding.mybmw
-```
+You can restrict the accounts and vehicles for the fingerprint generation.
+Full syntax is available through the `mybmw help` console command.
 
-The debug fingerprint is generated every time the discovery is executed.
-To force a new fingerprint perform a _Scan_ for MyBMW things.
-Personal data is eliminated from the log entries so it should be possible to share them in public.
+Personal data is eliminated from fingerprints so it should be possible to share them in public.
 Data like
 
 - Vehicle Identification Number (VIN)
 - Location data
 
-are anonymized.
-You'll find the fingerprint in the logs with the command
+are anonymized in the JSON response and URL's.
 
-```shell
-grep "Discovery Fingerprint Data" openhab.log
-```
+After the corresponding fingerprint is generated please [follow the instructions to raise an issue](https://community.openhab.org/t/how-to-file-an-issue/68464) and attach the fingerprint!
 
-After the corresponding fingerprint is generated please [follow the instructions to raise an issue](https://community.openhab.org/t/how-to-file-an-issue/68464) and attach the fingerprint data!
 Your feedback is highly appreciated!
 
+#### Debug Logging
```

Comment:
Please remove this debug logging documentation as this is subject is allready documented in a generic way at  a centralized place.

Reply:
Hi @lsiepel , to be honest I don't understand what's the issue with that. There is a link to the general documentation and here it is just mentioned in detail what's the name of the current binding and it mentions that no personal data is logged... this is imho an important information to the user...

Reply:
In general it is best to (if needed) point to the generic documentation to prevent duplicate information that is hard to maintain. If the logging command or other details would ever change, you don’t want to correct 200+ binding readme’s.

So I think it is mainly the example command here. I don’t have anything against the privacy comment.

Reply:
ok, fixed it.